src/model.ts

```diff
@@ -137,6 +137,46 @@ export function getDefaultName(prop: Property) {
       return 'ax-titml';
     case Property.AXIS_CHARACTERWIDTH:
       return 'ax-cw';
+    case Property.LEGEND:
+      return 'leg';
```

Comment:
why does it has to be 3 characters? Doesn't just `l` work?   (We want this to be short!)